src/drivers/legos/imu_driver.rs

```diff
@@ -0,0 +1,180 @@
+use std::{error::Error, ffi::CString};
+
+use hidapi::HidDevice;
+use packed_struct::{types::SizedInteger, PackedStruct};
+
+use super::{
+    event::{Event, InertialEvent, InertialInput},
+    hid_report::{InertialDataReport, InputReportType},
+};
+
+// Hardware ID's
+pub const VID: u16 = 0x1a86;
+pub const XINPUT_PID: u16 = 0xe310;
+pub const DINPUT_PID: u16 = 0xe311;
+pub const PIDS: [u16; 2] = [XINPUT_PID, DINPUT_PID];
+pub const IID: i32 = 0x5;
+
+// Input report sizes
+const INERTIAL_PACKET_SIZE: usize = 9;
+const HID_TIMEOUT: i32 = 10;
+
+// Input report axis ranges
+pub const GYRO_SCALE: i16 = 2;
```

Comment:
Is there any information about this scale value?

Reply:
IIRC it was added to fix the deg/s to more closely match the Deck. It's been a while and in this case was just moved.